### ISSUE TITLE
Injecting angular services in cypress tests

### ISSUE THREAD

**Issue by rs-maker** [cypress-io/cypress-example-recipes#698]
Hi i am currently looking to implement cypress e2e testing for a project I am contributing in https://github.com/ls1intum/Artemis
but i am in a very early stage and am having the same issue as described here: https://stackoverflow.com/questions/66591581/how-to-inject-angular-service-for-login-in-cypress-tests?newreg=5c602757587341b7aa95c4dbe2c5c018 (the solution in stack overflow is unfortunately not suitable)
Basically our services are supposed to be injected and i cant find a way to use them in my custom cypress commands. Does this mean i will not be able to log in using app code?
